src/lib/containers/entity_finder/EntityFinder.tsx

```diff
@@ -314,74 +272,92 @@ export const EntityFinder: React.FunctionComponent<EntityFinderProps> = ({
               showVersionSelection={showVersionSelection}
               selectColumnType={selectMultiple ? 'checkbox' : 'radio'}
               selected={selectedEntities}
-              includeTypes={showTypes}
+              visibleTypes={visibleTypesInList}
               selectableTypes={selectableTypes}
               toggleSelection={toggleSelection}
             />
           )}
           {
             <div style={searchActive ? { display: 'none' } : {}}>
-              <div className="EntityViewReflexContainer">
-                <SizeMe>
-                  {({ size }) => (
-                    <ReflexContainer
-                      key={(!!size.width).toString()}
-                      orientation="vertical"
-                      windowResizeAware
-                    >
-                      <ReflexElement minSize={200} size={350}>
-                        <TreeView
-                          sessionToken={sessionToken}
-                          setDetailsViewConfiguration={setConfigFromTreeView}
-                          showDropdown={true}
-                          visibleTypes={visibleTypesInTree}
-                          initialScope={initialScope}
-                          initialContainerId={initialContainerId}
-                        />
-                      </ReflexElement>
-                      <ReflexSplitter></ReflexSplitter>
-                      <ReflexElement minSize={400}>
-                        <EntityDetailsList
-                          sessionToken={sessionToken}
-                          configuration={configFromTreeView}
-                          showVersionSelection={showVersionSelection}
-                          selected={selectedEntities}
-                          includeTypes={showTypes}
-                          selectableTypes={selectableTypes}
-                          selectColumnType={
-                            selectMultiple ? 'checkbox' : 'radio'
-                          }
-                          toggleSelection={toggleSelection}
-                        />
-                      </ReflexElement>
-                    </ReflexContainer>
-                  )}
-                </SizeMe>
-              </div>
+              {treeOnly ? (
+                <div style={{ width: '550px', margin: 'auto' }}>
+                  <TreeView
+                    sessionToken={sessionToken}
+                    toggleSelection={toggleSelection}
```

Comment:
`toggleSelection` is only passed into the Tree component when `treeOnly` is true, allowing the component to add/remove selected entities